src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1074,6 +1090,12 @@ public static void mutate(List<? extends IMutation> mutations,
                     responseHandlers.add(mutateCounter((CounterMutation)mutation, localDataCenter, queryStartNanoTime));
                 else
                     responseHandlers.add(mutator.mutateStandard((Mutation)mutation, consistencyLevel, localDataCenter, standardWritePerformer, null, plainWriteType, queryStartNanoTime));
+
+                for (PartitionUpdate pu: mutation.getPartitionUpdates())
```

Comment:
Shouldn't this be executed before the `mutate` calls just above?

Reply:
7a9c72f23d89a2918f7852a3c928b53f0f1f4d59